tool-plugins/vscode/pom.xml

```diff
@@ -69,6 +158,51 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.google.code.maven-replacer-plugin</groupId>
+                <artifactId>replacer</artifactId>
+                <version>1.5.3</version>
```

Comment:
Use a property instead of having the version here.

Reply:
Sure. Will update.

Reply:
@Manuri, Done in a5c8359b1a26

Reply:
I'll merge. If anything else is to be done I'll send a follow up PR